core/src/main/scala/com/salesforce/op/stages/impl/classification/BinaryClassificationModelSelector.scala

```diff
@@ -206,7 +206,9 @@ case object BinaryClassificationModelSelector {
   ): ModelSelector[ModelType, EstimatorType] = {
     val modelStrings = modelTypesToUse.map(_.entryName)
     val modelsToUse =
-      if (modelsAndParameters == defaultModelsAndParams || modelTypesToUse != modelNames) modelsAndParameters
+      if (modelsAndParameters.isEmpty) defaultModelsAndParams
+        .filter{ case (e, p) => modelStrings.contains(e.getClass.getSimpleName) }
+      else if (modelTypesToUse != modelNames) modelsAndParameters
```

Comment:
if user only passes `modelTypesToUse = Seq(OpLogisticRegression)` the model selector wont work, since `modelsAndParameters` will be initialized as `Seq.empty`

Reply:
not true - if the user only passes modelTypesToUse = Seq(OpLogisticRegression) the first condition will be hit